app/assets/stylesheets/components/page_specific/_category_link_list.scss

```diff
@@ -27,6 +27,10 @@
     // This is to override the highly specific .editorial list styling
     content: none !important;
   }
+
+  > .category-link-list__item__link {
+    text-decoration: none;
+  }
```

Comment:
A quick note on this, adding ``text-decoration: none`` directly to the ``.category-link-list__item__link`` class isnt specific enough and is overwritten by a more global style for links, nesting it within the parent class fixes this, in case you were wondering